src/common/libkvs/kvs_lookup.c

```diff
@@ -126,7 +134,8 @@ flux_future_t *flux_kvs_lookup (flux_t *h, int flags, const char *key)
         return NULL;
     if (!(ctx = alloc_ctx (h, flags, key)))
         return NULL;
-    if (flags & FLUX_KVS_WATCH)
+    if (flags & FLUX_KVS_WATCH
```

Comment:
Parens around the bitwise & operations are usually advised here.  (hmm, thought `-Werror` would flag this as potentially confusing).